spec/graphql/hyacinth_schema_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe HyacinthSchema, solr: true do
+  subject(:graphql_request) { HyacinthSchema.execute(query, context: context, variables: variables) }
```

Comment:
A question of interest here. I hadn't seen 'subject' in action before. I'm curious about the advantage it confers.

Reply:
Ah!  Good question, and I'm glad you caught that, thanks!

I didn't actually mean to use subject there.  That came from copying that `HyacinthSchema.execute` line from another test file.  In effect, `subject` acts the same here as `let` since it's a named subject, but I think it's actually better to avoid `subject` and go with `let` instead.  I'll push another commit to change that.  I found this StackOveflow post that explains more about `subject` vs. `let` and makes a pretty good case for avoiding the use of subject in most cases: https://stackoverflow.com/a/38459039